src/main/java/practice/Sachin/Program1_Fibonacci.java

```diff
@@ -0,0 +1,24 @@
+package practice.Sachin;
+
+import java.util.Scanner;
+
+public class Program1_Fibonacci {
+
+    // Fibonacci Series
+    public static void main(String[] args){
+        Scanner sr = new Scanner(System.in);
+        System.out.println("Enter the Nummber : ");
+        int n= sr.nextInt();
+
+        int a=0;
+        int b=1;
+        int c;
+
+        for (int i=1; i<=n; i++){
+            c=a+b;
+            a=b;
+            b=c;
+            System.out.println("print the fibonacci series : "+a);
```

Comment:
if you take system.out.println line below c=a+b then fibbo series prints from 0,1....